.github/workflows/build.yml

```
@@ -53,32 +40,28 @@ env:
 
 jobs:
   buildAndCheck:
     strategy:
       fail-fast: false
       matrix:
-        # JDK 21 needed by IntelliJ Platform 2026.* targets (S12) and unblocks the
-        # CodeQL job whose `matrix.java != '23'` gate (lines below) was dead code
-        # while the matrix had only a single JDK. Multi-JDK shake-out is a free win.
+        # JDK 21 needed by IntelliJ Platform 2026.* targets (S12). JDK 23 keeps
+        # early-warning coverage for newer daemon/toolchain behavior.
         java: [ '21', '23' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)